iceoryx_posh/source/popo/building_blocks/chunk_queue_popper.cpp

```diff
@@ -54,6 +54,16 @@ cxx::optional<mepoo::SharedChunk> ChunkQueuePopper::pop() noexcept
     }
 }
 
+bool ChunkQueuePopper::hasOverflown() noexcept
+{
+    if(getMembers()->m_queueHasOverflown)

```

Comment:
as it is an atomic default would be sequential consistent. Relaxed should be fine here and cheaper, or?

Reply:
Done